src/physics/cam/clubb_intr.F90

```diff
@@ -2796,12 +2799,15 @@ real(r8) :: inv_exner_clubb(pcols,pverp)     ! Inverse exner function consistent
       qctend(:ncol,:)=0._r8
       inctend(:ncol,:)=0._r8
  
-      call liquid_macro_tend(npccn(:ncol,top_lev:pver),state1%t(:ncol,top_lev:pver), &
-         state1%pmid(:ncol,top_lev:pver),state1%q(:ncol,top_lev:pver,ixq),state1%q(:ncol,top_lev:pver,ixcldliq),&
-         state1%q(:ncol,top_lev:pver,ixnumliq),latvap,hdtime,&
-         stend(:ncol,top_lev:pver),qvtend(:ncol,top_lev:pver),qctend(:ncol,top_lev:pver),&
-         inctend(:ncol,top_lev:pver))
- 
+      do k = top_lev, pver
+         do i = 1, ncol
+            call liquid_macro_tend(npccn(i,k),state1%t(i,k), state1%pmid(i,k),    &
```

Comment:
Do we want to create a vectorized version of liquid_macro_tend?

Reply:
Thanks John. Just push the vectorized version of liquid_macro_tend.
The liquid_macro_tend and ice_macro_tend belong to CLUBB so I am not sure whether we should change them in our MG3 work.